library.properties

```
@@ -2,10 +2,10 @@ name=PhysicsLabFirmware
 version=1.0.0
 author=Arduino
 maintainer=Arduino <info@arduino.cc>
-sentence=Firmware for the Physics Lab kit.
-paragraph=This library depends on the ArduinoBLE and Adafruit LSM9DS1 Library.
+sentence=Firmware for the [Arduino Science Kit Physics Lab](https://www.arduino.cc/education/science-kit).
```

```suggestion
sentence=Firmware for the [Arduino Science Kit Physics Lab](https://www.arduino.cc/education/science-kit).
```
- Restore non-breaking spaces back to true spaces.

---
FYI, you are using the Markdown link syntax here but this is not likely to be rendered as Markdown anywhere. The classic Arduino IDE has some level of undocumented support for HTML, while Arduino CLI and Arduino IDE 2.x don't support any markup in this field (they just display it as plain text as-is). I don't know what Arduino Web Editor does.

But the use of the Markdown syntax it's not necessarily wrong because even unrendered Markdown is very readable and intuitive. So the choice is yours on that. I just wanted to make sure you are aware of the situation.